cmake/build-info.cmake

```diff
@@ -42,6 +42,8 @@ endif()
 if(MSVC)
     set(BUILD_COMPILER "${CMAKE_C_COMPILER_ID} ${CMAKE_C_COMPILER_VERSION}")
     set(BUILD_TARGET ${CMAKE_VS_PLATFORM_NAME})
+    add_compile_options("$<$<COMPILE_LANGUAGE:C>:/utf-8>")
+    add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:/utf-8>")
```

Comment:
Btw, which source files require this flag? I am wondering because in `llama.cpp` we don't seem to need this.

Reply:
`whisper.cpp` requires this flag:

https://github.com/ggerganov/whisper.cpp/blob/46d07b9c8506246bea4f2939470acfbbc63ce675/src/whisper.cpp#L4849

There are such flags in `llama.cpp`:

https://github.com/ggerganov/llama.cpp/blob/d774ab3acc4fee41fbed6dbfc192b57d5f79f34b/CMakeLists.txt#L51